pykafka/client.py

```diff
@@ -59,8 +57,8 @@ def __init__(self,
             milliseconds) when reading responses for offset commit and
             offset fetch requests.
         :type offsets_channel_socket_timeout_ms: int
-        :param ignore_rdkafka: Don't use rdkafka, even if installed.
-        :type ignore_rdkafka: bool
+        :param ignore_rdkafka: Deprecated, and in fact ignored. Instead, you
```

Comment:
Know what, I thought about this some more and realized that it's pointless to just deprecate this since it's never done anything to begin with. This is the rare case where I'd actually prefer to break old code that uses this kwarg (if any such code exists) without incrementing the major version.